dev-itpro/developer/methods-auto/recordref/recordref-setview-method.md

```diff
@@ -30,23 +30,22 @@ An instance of the [RecordRef](recordref-data-type.md) data type.
 
 *String*  
 &emsp;Type: [Text](../text/text-data-type.md)  
-The string format is the same as the SourceTableView property on pages.  
-
+A string that contains the sort order, key, and filter to set. The string format is the same as the SourceTableView Property on pages.
```

Comment:
Why is property capitalized?

Reply:
> Why is property capitalized?

Side effect of being used to the German orthography ;-) Will be fixed soon ...